cirq-rigetti/requirements.txt

```diff
@@ -0,0 +1,303 @@
+attrs==20.3.0 \
```

Comment:
What do the hashes mean?

Reply:
For supporting, pip [hash checking mode](https://pip.pypa.io/en/stable/cli/pip_install/#hash-checking-mode). Those are output automatically by [poetry export](https://python-poetry.org/docs/cli/#export), but I've removed them for now.